docs/conf.py

```diff
@@ -45,8 +45,8 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'sphinx.ext.autodoc',
     'sphinx.ext.intersphinx',
+    'sphinx.ext.autodoc',
```

Comment:
Any specific reason behind this change?

Reply:
Oh, not really. I was trying to get intersphinx working and had moved some things around in the off chance that there was a side-effect in the ordering.